app/views/patron_requests/new.html.erb

```diff
@@ -60,16 +60,16 @@
           <table class="table mb-0 border">
             <thead>
               <tr>
-                <th class="text-start">Call number</th>
-                <th class="text-end">Status</th>
+                <th class="text-start" data-patronRequest-sortfield="Callnumber" data-patronRequest-asc=true data-action='click->patronRequest#sort'>Call number</th>
```

Comment:
Do we need to do something additional to make these accessible (e.g. a button, or add it to the tabindex) or add a pointer cursor?

Reply:
I think it is okay because of this? 

`Header cells must be marked up with <th>, and data cells with <td> to make tables accessible. For more complex tables, explicit associations may be needed using scope, id, and headers attributes.`

https://www.w3.org/WAI/tutorials/tables/

Reply:
Hm, does that account for interactivity?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-sort suggests buttons and an `aria-sort`?

Reply:
added button and aria-sort

Reply:
That page also says:
> The aria-sort attribute should only be added to a single table or grid header at a time. The attribute is set to inform assistive technology users which column or row is sorted.

Reply:
Oops. Thanks for the catch. Update now pushed.